jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml

```diff
@@ -162,15 +162,15 @@
     - kubernetes-node-e2e:
         max-total: 12
         job-name: pull-kubernetes-node-e2e
-        json: 0
+        json: 1
```

Comment:
Will go away as soon as any yaml file uses json mode for everything, then I'll just move it into the template.

Once all jobs use json mode I'll delete the flag.